ami/install.bash

```diff
@@ -42,68 +59,48 @@ apt-get install -y \
 # Download, compile and install Python 3.6.2
 wget https://www.python.org/ftp/python/3.6.2/Python-3.6.2.tgz
 tar xvf Python-3.6.2.tgz && cd Python-3.6.2/
-./configure --enable-optimizations
+./configure
 make -j
 sudo make install
 cd ../ && rm -r Python-3.6.2
+# Use this instead of the manual python compile when we switch to buster base image. Make sure we get Python3.6
+#echo -e "\n=== Install Python 3, Pip and Flake8 ==="
+#apt-get install python3-distutils python3 -y
```

Comment:
I think these can go.

Reply:
I've added them again so I don't have to figure out what packages to install when we switch to buster...